### ISSUE TITLE
Reset password incorrect response 

### ISSUE THREAD

**Issue by yarreg** [apragacz/django-rest-registration#110]
When changing password with API endpoint: POST /api/accounts/change-password/
If the password does not match the specified rule response is:
```
{"password":["This password is too short. It must contain at least 8 characters."]}
```

However, when resetting password response is:
POST api/accounts/reset-password/
```
["This password is too short. It must contain at least 8 characters."]
```

**Comment by apragacz** [comment]
@yarreg I'm not 100% sure what you exaclty meant. Did you mean that for reset password the response should be:
```
{"password": ["This password is too short. It must contain at least 8 characters."]}
```
instead of:
```
["This password is too short. It must contain at least 8 characters."]
```
?

Also, I'd recomment using predefined issue templates in the future:
https://github.com/apragacz/django-rest-registration/issues/new/choose
There is one for bug report already. Issues following the template have the highest priority as It's much easier for me to understand what the real issue is.

**Comment by yarreg** [comment]
@apragacz Yes that's right

**Comment by apragacz** [comment]
This fix will be released in next version (`0.6.0`).
